services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/validation.py

```diff
@@ -260,19 +262,16 @@ async def validate_compose_spec(
     )
 
     # validate against docker-compose config
-
-    command = "docker-compose --file {file_path} config"
-    finished_without_errors, stdout = await write_file_and_run_command(
+    finished_without_errors, stdout = await docker_compose_config(
+        compose_spec,
         settings=settings,
-        file_content=compose_spec,
-        command=command,
-        command_timeout=None,
+        command_timeout=docker_compose_config_timeout,
```

Comment:
I will add this to an env-var so it can be tuned. 
IMO it should not be ``None`` though

Reply:
Added as a query parameter in docker-compose up and down

Reply:
I've checked inside the dynamic scheduler and if this call fails it will get called over and over again until it manages to get it through.
Let's see what happens now under the same conditions.

Reply:
Adjusted timeouts with @GitHK